cloudify_aws/kms/tests/test_alias.py

```diff
@@ -60,7 +64,7 @@ def test_create_raises_UnknownServiceError(self):
             alias.create(ctx=_ctx, resource_config=None, iface=None)
 
         self.assertEqual(
-            str(error.exception),
+            text_type(error.exception),
```

Comment:
Same question. No change is necessary, just curious.

Reply:
Same as above